src/averageAges.js

```diff
@@ -20,6 +20,25 @@ function calculateMenAverageAge(people, century) {
   // avoid using loop and forEach
   // replace `if ()` statement with &&, || or ?:
   // without nesting
+
+  const manCheck = person => person.sex === 'm';
+  const centuryCheck = person => {
+    return manCheck(person) && Math.ceil(person.died / 100) === century;
+  };
+
+  let men;
+
+  if (arguments.length === 2) {
+    men = people.filter(centuryCheck);
+  } else {
+    men = people.filter(manCheck);
```

Comment:
Better use ternary operator for that:
```
const men = condition
  ? .....
  : .....
```
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Conditional_operator